docs/modules/ROOT/pages/reference/kube-apis/application-api.adoc

```diff
@@ -0,0 +1,680 @@
+// Generated documentation. Please do not edit.
+:anchor_prefix: k8s-api
+
+[id="reference"]
+== Application API Reference
+
+.Packages
+- xref:{anchor_prefix}-appstudio-redhat-com-v1alpha1[$$appstudio.redhat.com/v1alpha1$$]
+
+
+[id="{anchor_prefix}-appstudio-redhat-com-v1alpha1"]
+=== appstudio.redhat.com/v1alpha1
+
+Package v1alpha1 contains API Schema definitions for the appstudio v1alpha1 API group
+
+.Resource Types
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-application[$$Application$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-applicationlist[$$ApplicationList$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-component[$$Component$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-componentdetectionquery[$$ComponentDetectionQuery$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-componentdetectionquerylist[$$ComponentDetectionQueryList$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-componentlist[$$ComponentList$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-snapshot[$$Snapshot$$]
+- xref:{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-snapshotlist[$$SnapshotList$$]
+
+
+
+[id="{anchor_prefix}-github-com-konflux-ci-application-api-api-v1alpha1-application"]
+==== Application
+
+
+
+Application is the Schema for the applications API.  For a detailed description with examples, refer to <a href="https://github.com/redhat-appstudio/book/blob/main/book/HAS/hybrid-application-service-api.md"> Hybrid Application Service Kube API </a>
```

Comment:
This is not proper adoc link
![image](https://github.com/user-attachments/assets/1b8fdfb1-9116-4bd9-a1a9-0468fef45d00)

Reply:
Agreed. Since this is auto-generated content, I'd consider fixing this out of scope in this PR. I plan on issuing a "call to action" for teams that have their API docs auto-generated.